content/millennium/r4/care-provision/care-team.md

```diff
@@ -35,7 +35,7 @@ Search for CareTeams that meet supplied query parameters:
 
 _Implementation Notes_
 
-* For CareTeam Search operations, both Careteam.read and Encounter.read scopes are required.
+* For CareTeam Read operations, only Careteam.read is required.
```

Comment:
It should be `For CareTeam Search operations`

Reply:
changes addressed